externals/kyuubi-flink-sql-engine/bin/flink-sql-engine.sh

```diff
@@ -37,31 +37,26 @@ FLINK_SQL_ENGINE_HOME="$(cd `dirname $0`/..; pwd)"
 if [[ "$FLINK_SQL_ENGINE_HOME" == "$KYUUBI_HOME/externals/engines/flink" ]]; then
   FLINK_SQL_ENGINE_LIB_DIR="$FLINK_SQL_ENGINE_HOME/lib"
   FLINK_SQL_ENGINE_JAR=$(find "$FLINK_SQL_ENGINE_LIB_DIR" -regex ".*/kyuubi-flink-sql-engine_.*\.jar")
-  FLINK_HADOOP_CLASSPATH="$INTERNAL_HADOOP_CLASSPATHS"
```

Comment:
Do we need this one? What if user provides the hadoop jar?

Reply:
@pan3793, the user should configure the `HADOOP_CLASSPATH` or `HADOOP_CONF_DIR` or `YARN_CONF_DIR`
because INTERNAL_HADOOP_CLASSPATHS=${HADOOP_CLASSPATH}:${HADOOP_CONF_DIR}:${YARN_CONF_DIR}

Reply:
Then after removing this, Kyuubi can not see Hadoop classes even user provides them?